autodoc.pl

```diff
@@ -470,8 +469,7 @@
                             'vutil.c' => $versioning_scn,
                            );
 
-sub where_from_string($;$) {
-    my ($file, $line_num) = @_;
+sub where_from_string($file, $line_num = 0) {
```

Comment:
Stylistically, I find it best to put a space between the function name and its signature parens, because such a space is not permitted on attributes. It keeps it looking more visually distinct; much like the others in this file.

Reply:
ok